Calculator/README.md

```diff
@@ -0,0 +1,48 @@
+# Calculator
+
+This sample is built with Jetpack Compose, the new UI framework in Android.
+
+## Getting Started
+
+To learn how to load apps on the Surface Duo emulator, see the [documentation](https://docs.microsoft.com/dual-screen/android), and follow [the blog](https://devblogs.microsoft.com/surface-duo).
+
+## Features
+
+The sample uses the [Companion Pane](https://docs.microsoft.com/dual-screen/introduction#companion-pane) app pattern to show a basic Calculator on one pane, and additional more advanced grid of equations on the other pane. This sample also has a ticker tape, which is a history of entered equations that can be accessed later. Depending on the orientation, this ticker tape will be on the first or second pane.
```

Comment:
I may argue that this sample is using `Extended Canvas`, rather than `Companion Pane`.